env/local/.env.sh

```diff
@@ -0,0 +1,5 @@
+export STREAMX_TOKEN_PIM=
```

Comment:
local should be default from .env file, that is overridden by sh script for cloud deployment.
https://quarkus.io/guides/images/config-sources.png